ci/dib_image/dib-and-image-vars.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eux
+
+current_dir="$(dirname "$(readlink -f "${0}")")"
```

Comment:
```suggestion
CURRENT_DIR="$(dirname "$(readlink -f "${0}")")"
```
Rule of thumb: Global vars in UPPER case, local variables lower case.